Ensure screen stays on during say all (#17651)
nvaccess/nvda: user_docs/en/changes.md

* NVDA can now be configured to speak the current line or paragraph when navigating with braille navigation keys. (#17053, @nvdaes)
* In Word, the selection update is now reported when using Word commands to extend or reduce the selection (`f8` or `shift+f8`). (#3293, @CyrilleB79)
* In Microsoft Word 16.0.18226 and higher or when using Word object model, NVDA will now report if a heading is collapsed in both speech and braille. (#17499)

> * Added an advanced setting to prevent system lock during say all or reading with Braille. (#17649, @LeonarddeR)

Reviewer: ```suggestion
* Added a general setting to prevent system lock during say all or reading with braille. (#17649, @LeonarddeR)
```
